src/h_vialib/client.py

```diff
@@ -13,7 +13,7 @@
 class ContentType(str, Enum):
     PDF = "pdf"
     HTML = "html"
-    VIDEO = "video"
+    YOUTUBE = "youtube"
```

Comment:
This'll require an update in Via when it upgrades to the next version of h-vialib: changing `ContentType.VIDEO` to `ContentType.YOUTUBE`